src/MoBi.R/Services/SimulationCreateTask.cs

```diff
@@ -0,0 +1,23 @@
+﻿using MoBi.R.Domain;
+using OSPSuite.R.Domain;
+
+namespace MoBi.R.Services
+{
+   public interface ISimulationCreateTask
+   {
+      Simulation CreateSimulationFrom(SimulationConfiguration simulationConfiguration);
+   }
+
+   public class SimulationCreateTask : ISimulationCreateTask
```

Comment:
I would just call it `SimulationTask`. I bet there will be more operations that will be done with simulations.